app/views/lessons/installation_lessons/index.html.erb

```diff
@@ -1,14 +1,20 @@
-<%= title('Installation Lessons Appendix') %>
+<%= title('Installation Guides') %>
 
-<div class="page-container h-screen">
-  <h1 class="page-heading-title text-center">Installation Lessons Appendix</h1>
+<div class="page-container min-h-screen">
+  <h1 class="page-heading-title text-center">Installation Guides</h1>
   <ul>
     <% @lessons.each do |lesson| %>
-       <li>
-          <p class="text-1xl text-center m-3 text-gray-600 dark:text-gray-300">
-            <%= link_to lesson.title, lesson_path(lesson), class: 'underline hover:no-underline hover:text-gray-500 dark:hover:text-gray-400' %>
-          </p>
-        </li>
+      <li class="max-w-md mx-auto my-4 rounded-lg shadow transition bg-white hover:bg-gray-100 dark:bg-gray-800 dark:hover:bg-gray-700">
```

Comment:
This is a lot like our card styling. We can reuse that here by using our CardCompent to make sure everything stays consistent 💪 

We're better off changing this from an unordered list and just have items within a containing section element which will handle the max width and spacing between the elements:
```erb
<section class="max-w-xl mx-auto space-y-5">
    <% @lessons.each do |lesson| %>
      <%= render CardComponent.new(classes: "transistion dark:hover:bg-gray-700 hover:bg-gray-100") do |card| %>
        <% card.body(classes: "py-4 px-4") do %>
          <%= link_to lesson_path(lesson) do %>
            <div class="flex items-center gap-3">
              <%= inline_svg "icons/book.svg", alt: 'lesson icon', class: 'w-6 dark:text-gray-500 text-gray-400 shrink-0', title: 'Lesson' %>

                <p class=" text-gray-600 dark:text-gray-300">
                  <%= lesson.title %>
                </p>
            </div>
          <% end %>
        <% end %>
      <% end %>
    <% end %>
  </section>
```

Reply:
Thanks for the feedback, @KevinMulhern! I'll try to have the changes made by the end of the week.

Reply:
I've made the proposed changes.